phongo_compat.h

```diff
@@ -206,6 +182,35 @@
 # define PHP_STREAM_CONTEXT(stream) ((php_stream_context*) (stream)->context)
 #endif
 
+#if SIZEOF_PHONGO_LONG == 8
+# define ADD_INDEX_INT64(zval, index, value) add_index_long(zval, index, value)
+# define ADD_ASSOC_INT64(zval, key, value) add_assoc_long(zval, key, value)
```

Comment:
These definitions were moved up to be consistent with `bson.c`, where we check for 64-bit platforms first. Additionally, I removed the trailing semi-colon from `ADD_ASSOC_INT64()`'s definition for consistency.